common-app-lib/resources/templates/structure.html

```diff
@@ -13,6 +13,7 @@
     <link href="https://cdn.earthdata.nasa.gov/eui/1.1.3/stylesheets/application.css" rel="stylesheet" />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,300" rel="stylesheet" type="text/css">
     <link href="https://cdn.earthdata.nasa.gov/eui/1.1.3/docs/assets/docs.css" rel="stylesheet" />
```

Comment:
For now, I've copied the remote `docs.css` from the style guide to a local `eui.css` stylesheet that CMR will serve.